pkg/cli/cli.go

```diff
@@ -1194,16 +1195,31 @@ func buildServiceBuilder(builder client.ServiceBuilder, authnConfig *AuthnConfig
 			RetryDelay:     5 * time.Second,
 			ChunkSizeBytes: chunkSizeBytes,
 		}
-		if authnConfig != nil && authnConfig.OIDCToken != nil {
-			content, err := os.ReadFile(authnConfig.OIDCToken.TokenFile)
-			if err != nil {
-				return nil, fmt.Errorf("failed loading oidc token: %w", err)
-			}
-			value := strings.TrimSuffix(string(content), "\n")
-			opts.Authn = &client.AuthnOpts{
-				OIDCToken: &client.OIDCToken{
+		if authnConfig != nil {
+			opts.Authn = &client.AuthnOpts{}
+			if authnConfig.OIDCToken != nil {
```

Comment:
we can add validation here as well to make sure only one authentication option is used in the configuration.

Reply:
Done.